docs/content/concepts/sui-move-concepts/conventions.mdx

```diff
@@ -1,494 +1,458 @@
+module conventions::wallet;
 
-    // ❌ Wrong
-    struct RequestPotato {}
+public struct Wallet has key, store {
+    id: UID,
+    amount: u64
+}
+
+
+module conventions::claw_back_wallet;
+
+public struct Wallet has key {
+    id: UID,
+    amount: u64
 }
 ```
 
-## Read functions
+### Body
 
-Be mindful of the dot syntax when naming functions. Avoid using the object name on function names.
+Structure your code using comments to create sections for your Move code files. Structure your titles using `===` on either side of the title.
 
 ```move
-module conventions::profile {
+module conventions::comments;
 
-    struct Profile {
-        age: u64
-    }
+// === Imports ===
 
-    // ✅ Right
-    public fun age(self: &Profile): u64 {
-        self.age
-    }
+// === Errors ===
 
-    // ❌ Wrong
-    public fun profile_age(self: &Profile): u64 {
-        self.age
-    }
-}
+// === Constants ===
 
-module conventions::defi {
+// === Structs ===
 
-    use conventions::profile::{Self, Profile};
+// === Events ===
 
-    public fun get_tokens(profile: &Profile) {
+// === Method Aliases ===
 
-     // ✅ Right
-     let name = profile.age();
+// === Public Functions ===
 
-     // ❌ Wrong
-     let name2 = profile.profile_age();
-    }
-}
+// === View Functions ===
+
+// === Admin Functions ===
+
+// === Package Functions ===
+
+// === Private Functions ===
+
+// === Test Functions ===
 ```
 
-## Empty function
+Here, "Public Functions" are the functions modifying state, "View Functions" are often on-chain getters or off-chain helpers. The latter are not necessary because data is easily readable by querying objects. `init` function should be the first function in the module if it exists.
 
-Name the functions that create data structures as `empty`.
+Try to sort your functions by their purpose and according to the user flow to improve readability, you can also use explicit function names like `admin_set_fees` to make it clear what the function does.
 
-```move
-module conventions::collection {
+Ideally, test functions should only consists of `[test_only]` helpers for the actual tests that should be located in the tests directory.
 
-    struct Collection has copy, drop, store {
-        bits: vector<u8>
-    }
+Imports can be grouped by dependency, for example:
 
-    public fun empty(): Collection {
-        Collection {
-            bits: vector[]
-        }
-    }
-}
+```move
+use std::string::String;
+use sui::{
+    coin::Coin,
+    balance,
+    table::Table
+};
+use my_dep::battle::{Battle, Score};
 ```
 
-## New function
+## Naming Conventions
+
+### Constants
 
-Name the functions that create objects as `new`.
+Constants should be in uppercase and use snake case. Errors are specific constants that use PascalCase and start with an E. Make them descriptive.
 
 ```move
-module conventions::object {
+module conventions::constants;
 
-    use sui::object::{Self, UID};
-    use sui::tx_context::TxContext;
+const MAX_NAME_LENGTH: u64 = 64;
```

Comment:
Might be useful to add a comment here clarifying that this is a non-error constant (so upper snake case good), and that the two examples below are error consts